backend/src/apiserver/resource/resource_manager.go

```diff
@@ -1012,6 +1012,23 @@ func (r *ResourceManager) CreateJob(ctx context.Context, job *model.Job) (*model
 	return r.jobStore.CreateJob(job)
 }
 
+func (r *ResourceManager) getOwnerReferences() []v1.OwnerReference {
+	dspaName := common.GetStringConfigWithDefault("DSPA_NAME", "")
+
+	if dspaName == "" {
+		return []v1.OwnerReference{}
+	} else {
+		return []v1.OwnerReference{
+			{
+				APIVersion: common.GetStringConfig("DSPA_API_VERSION"),
+				Kind:       common.GetStringConfig("DSPA_KIND"),
+				Name:       dspaName,
+				UID:        types.UID(common.GetStringConfig("DSPA_UID")),
+			},
```

Comment:
Can we not mention DSPA, and keep this DSP-agnostic? I suggest "OWNER_NAME", "OWNER_API_VERSION", etc.

Reply:
Done.